### DISCUSSION TITLE
URL "zxing.net" redirects to a page from a competitor

### DISCUSSION THREAD

**Discussion by HansHappyHorizon** [micjahn/ZXing.Net#573] [General]
If you navigate to [zxing.net](https://zxing.net/) you'll notice it redirects to a page from IronSoftware that tries to convince you to choose their IronBarcode software instead of ZXing.Net. If you Google "zxing.net" it even lists that page as the first result. This is somewhat misleading.

I know it's not uncommon for companies to try to convince you to use their software instead of a competitor's, even directly comparing against existing software, so I don't think it's worth it to try to get that page taken down. But to claim the domain name of the competitor (zxing.net) and having that redirect to their own page seems a bit extreme to me. I'm no lawyer, so I have no idea how legal this is.

Can any action be taken to claim the "zxing.net" domain? I think there's a fair shot, considering it's directly linked to the name of this package. Unless, of course, there is no desire for this, in which case this discussion can be closed.

**Comment by micjahn** [comment]
Thanks for reporting that issue here.
However, I'm afraid I don't have the time, money or nerves to deal with this company regarding the transfer of the domain. In particular, I wish I had enough free time to do this, but time is so short that I have not yet been able to prepare a new version of zxing.net. And I think that would be more important.
But thanks again for pointing me there.
